examples/groth16-verifier/Cargo.toml

```
@@ -0,0 +1,16 @@
+[package]
+name = "groth16-verifier"
```

Cargo treats package/crate names as part of a global namespace. Adding `risc0` in front should help avoid any collisions. (We currently have an issue that spits out warnings sometimes related to the xg boost example related to this)
```suggestion
name = "risc0-groth16-verifier-example"
```
This suggestion is verbose, but also no one will import this crate 🤷